content/sips/sip-192.md

```diff
@@ -0,0 +1,61 @@
+---
+sip: 192
+network: L1 + l2
+title: Fee Pool Accounting Error Fix and Reimbursement
+status: Draft
+author: Daniel Beal (@dbeal-eth)
+implementor: Daniel Beal (@dbeal-eth)
+release: 12-2-2021
+discussions-to: 'https://research.synthetix.io'
+created: 2021-11-18T00:00:00.000Z
+type: Bug
+---
+
+## Simple Summary
+
+Fix an error where fees from Wrappers are double-counted, and repay the difference to match fee pool balance to 
+
+## Abstract
+
+<!--A short (~200 word) description of the proposed change, the abstract should clearly describe the proposed change. This is what *will* be done if the SIP is implemented, not *why* it should be done or *how* it will be done. If the SIP proposes deploying a new contract, write, "We propose to deploy a new contract that will do x".-->
+
+There was an error in the WrapperFactory which has caused the amount of fees collected to be double-counted. This leads to stakers claiming a higher amount
+than they should be able to, resulting in a loss of funds in the fee payment address. 
+
+## Motivation
+
+<!--This is the problem statement. This is the *why* of the SIP. It should clearly explain *why* the current state of the protocol is inadequate.  It is critical that you explain *why* the change is needed, if the SIP proposes changing how something is calculated, you must address *why* the current calculation is inaccurate or wrong. This is not the place to describe how the SIP will address the issue!-->
+
+Stakers are not able to claim when the fee address lacks the funds necessary, so we need to take action to restore full functioning of the system.
+
+## Specification
+
+<!--The specification should describe the syntax and semantics of any new feature, there are five sections
+1. Overview
+2. Rationale
+3. Technical Specification
+4. Test Cases
+5. Configurable Values
+-->
+
+### Overview
+
+The missing funds (currently ~$257k) will be funded to the Synthteix fee address on L2 to make up for the shortfall in fees, and to allow stakers to continue
```

Comment:
This part isn't required in the SIP - it's separate to the changes to the protocol.